.circleci/config.yml

```diff
@@ -866,6 +866,30 @@ jobs:
           command: yarn type:doc
       - store_artifacts:
           path: ./docs/out
+  
+  # Node core language tests
+
+  node-core-language-tests:
+    resource_class: small
+    working_directory: ~/dd-trace-js
+    docker:
+      - image: cimg/python:3.9.4-node
+      - image: kyleverhoog/dd-trace-test-agent:latest
```

Comment:
Right now there is already a test agent in this repository. I don't know whether it would make sense to start using this one instead, but maybe it makes sense for integration tests.

@bengl Thoughts?

Reply:
Is validating correct traces even necessary, when we have tests for exactly this?

Reply:
I'm not sure about what tests are currently running, so the checks the test-agent makes might be redundant.
But I don't think having another harness of safety is bad, and having a library agnostic tool that does the same checks across multiple tracers could be useful to avoid multiplying efforts by the number of libraries, and ensuring they behave the same.

Reply:
I would say for now to leave this out of this PR and it can be discussed later. There are a lot of subtleties about validating traces, and in any case the goal here is to make sure we don't break the libraries, not to make sure we are sending valid traces. This can be done in a future PR.